astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -188,7 +188,7 @@ def make_transform_graph_docs(transform_graph):
     docstr = dedent(docstr) + "        " + graphstr.replace("\n", "\n        ")
 
     # colors are in dictionary at the bottom of transformations.py
```

Comment:
This comment is not right any more...  But good to just change the import!

Reply:
Changed.